swagger.yaml

```diff
@@ -2595,6 +2596,23 @@ definitions:
       type: string
       pattern: '^([0-9]{1,3}\.){3}[0-9]{1,3}\/[0-9]|[1-2][0-9]|3[0-2]?$'
 
+  connectivity_check_api_request:
+    type: object
+    required:
+      - scheme
+      - host
+      - port
+    properties:
```

Comment:
Do you really want it broken into scheme, host, and port, or would it make more sense to have a single string with all 3?  (Just asking, I don't know which is better for your case)

Reply:
Agree.  A single URL string

Reply:
ok, changed to a single URL